test/unit/Conversions.spec.ts

```diff
@@ -1,31 +1,37 @@
+import { RelayPricer } from '@rsksmart/rif-relay-client';
 import BigNumber from 'bignumber.js';
 import { expect, use } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import Sinon from 'sinon';
 import sinonChai from 'sinon-chai';
 import {
     getXRateFor,
-    RBTC_IN_RIF,
     toNativeWeiFrom,
     toPrecision
 } from '../../src/Conversions';
-import Token from '../../src/definitions/token.type';
+import ExchangeToken from '../../src/definitions/token.type';
 
 use(sinonChai);
 use(chaiAsPromised);
 
 describe('Conversions', () => {
+    const xRateRifRbtc = BigNumber('0.00000332344907316948');
+
     afterEach(() => {
         Sinon.restore();
     });
 
     describe('getXRateFor', async () => {
```

Comment:
I think this function require testing alternative scenarios

Reply:
updated